Wrapping/Generators/Python/itk/pyi_generator.py

```diff
@@ -0,0 +1,590 @@
+                    or base.typed != class_def.typed
+                    or base.is_abstract != class_def.is_abstract
+                    or base.is_enum != class_def.is_enum
+                    or base.has_superclass != class_def.has_superclass
+            ):
+                raise AssertionError(
+                    f"The basic values of two associated classes do not match.\n"
+                    f"Item 1:"
+                    f" class `{base.submodule_name}::{base.class_name}`"
+                    f" has_new_method: `{base.has_new_method}`"
+                    f" typed: `{base.typed}`"
+                    f" is_abstract: `{base.is_abstract}`"
+                    f" is_enum: `{base.is_enum}`"
+                    f" has_superclass: `{base.has_superclass}`"
+                    f"Item 2:"
+                    f" class `{class_def.submodule_name}::{class_def.class_name}`"
+                    f" has_new_method: `{class_def.has_new_method}`"
+                    f" typed: `{class_def.typed}`"
+                    f" is_abstract: `{class_def.is_abstract}`"
+                    f" is_enum: `{class_def.is_enum}`"
+                    f" has_superclass: `{class_def.has_superclass}`"
+                )
+
+            # Enums should not be defined in multiple submodules
+            # If this is not the case, a combine function could be added here
+            if base.is_enum:
+                raise TypeError(
+                    f"The enumeration class {base.class_name} is defined in multiple ITK submodules.\n"
+                    f"If this is"
+                )
+
+            # In some instances, not all parent classes are represented in each overload
+            # This merges them together but may provide some type dependant hints
+            base.parent_class = list(set(base.parent_class + class_def.parent_class))
+
+            # merge class methods together
+            for (
+                    method_name,
+                    method_overloads,
+            ) in class_def.python_method_headers.items():
+                if method_name in base.python_method_headers:
+                    # Add overloads that don't already exist in the base method's overload list
+                    for overload in method_overloads:
+                        if overload not in base.python_method_headers[method_name]:
+                            base.python_method_headers[method_name].append(overload)
+                else:
+                    # In this case there is a method that is only defined for some overloads of the class
+                    # The best we can do is always show it and let it error when the overload does not
+                    # contain the method
+                    base.python_method_headers[
+                        method_name
+                    ] = class_def.python_method_headers[method_name]
+        return base
+
+
+# Press the green button in the gutter to run the script.
```

Comment:
I did not understand this comment.

Reply:
I think @hjmjohnson must have added this? 
But I think it is referring to the start button in the Pycharm IDE:
![image](https://user-images.githubusercontent.com/55513580/156456393-cdbda00d-c08a-4713-803d-d422bb251607.png)